extensions/binding/org.eclipse.smarthome.binding.weatherunderground/src/main/java/org/eclipse/smarthome/binding/weatherunderground/WeatherUndergroundBindingConstants.java

```diff
@@ -12,13 +12,19 @@
  */
 package org.eclipse.smarthome.binding.weatherunderground;
 
+import java.util.Collections;
+import java.util.Set;
+
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
 
+import com.google.common.collect.Sets;
```

Comment:
We try to avoid using guava in new contributions, see https://www.eclipse.org/smarthome/documentation/development/guidelines.html#c-language-levels-and-libraries